src/orgChart/customNodes/EldorNodes.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { memo } from "react";
+import { Handle } from "react-flow-renderer";
+import eldor from './images/eldor.png';
+import "../OrgChart.css";
+
+export default memo(() => {
+  return (
+    <div className={"branch-box"}>
+      <div>
+        <img className={"image"} src={eldor} alt={"eldor"} />
+      </div>
+      <div>{"Eldor Ergashov"}</div>
+      <Handle type="target" position="top" />
+    </div>
+  );
+});
```

Comment:
DRY

Reply:
Ismva rasmlarni to'g'ridan-to'g'ri componentga qo'ygandan ko'ra, `data` prop ichiga qo'yganingiz yaxshi edi. Namuna: https://reactflow.dev/examples/custom-node/